x-pack/qa/rolling-upgrade/src/test/resources/rest-api-spec/test/mixed_cluster/80_transform_jobs_crud.yml

```diff
@@ -1,12 +1,17 @@
 ---
 "Test put batch transform on mixed cluster":
+  - skip:
+      features: allowed_warnings
+
   - do:
       cluster.health:
         index: "transform-airline-data"
         wait_for_status: green
         timeout: 70s
 
   - do:
+      allowed_warnings:
```

Comment:
There really are a lot of these! Maybe it'd be nicer to allow warnings on a test case level rather than on a "do" level, but I like precision if I can get it and adding `allowed_warnings` on the do level really was simpler to implement.